src/PillLink/PillLink.tsx

```diff
@@ -0,0 +1,34 @@
+import clsx from "clsx";
+import React from "react";
+
+import useStyles from "./styles";
+
+export interface PillLinkProps
+  extends React.DetailedHTMLProps<
+    React.AnchorHTMLAttributes<HTMLAnchorElement>,
+    HTMLAnchorElement
+  > {
+  state?: "default" | "hover" | "active";
```

Comment:
Is there a case where we want to force active or hover state without user interaction?